extensions/amp-list/0.1/amp-list.js

```diff
@@ -551,6 +587,14 @@ export class AmpList extends AMP.BaseElement {
           });
         }
         removeChildren(dev().assertElement(this.container_));
+      };
+
+      if (!this.loadMoreEnabled_ && this.enableManagedResizing_) {
```

Comment:
Duped in a few places so I'd consider making this a helper method `allowManagedResize()`.

Reply:
This is only present twice, in `resetIfNecessary_` and `render_`. Would it still be helpful to make a new method for those instances? Other conditional blocks evaluate other combinations, i.e. `!X && !Y` or `!(X && Y)` as opposed to `!X && Y`

Reply:
Oh I see. A comment would do too.